### ISSUE TITLE
prefactor not taken into account when normalizing BT9 data to monitor rate

### ISSUE THREAD

**Issue by js54434** [neutronpy/neutronpy#20]
If we have two data sets data1 and data2 from BT9 data with the same monitor rate but different prefactors, then loading and adding the data in the following way doesn't seem to include the prefactor in the normalization:

```
data1 = Data()
data2 = Data()
data1.load_file(filename1, mode='ICP')
data2.load_file(filename2, mode='ICP')
data = data1+data2
```

**Comment by pseudocubic** [comment]
@js54434 Could you link me to two example files that you are trying to combine so that I can confirm that I handle this bug? Thanks!

**Comment by pseudocubic** [comment]
Addressed by 70280d763344124a438d312e1307b0e04a168dd9

**Comment by js54434** [comment]
Sorry for the delay. For me, the desired behavior is to be able to normalize to Mon*Prf. I can confirm that the current version of the code (0.2.0) does this.
